lib/ansible/modules/system/mount.py

```diff
@@ -33,7 +33,7 @@
     aliases: [ name ]
   src:
     description:
-      - Device to be mounted on I(path).
+      - Device (or NFS volume, or something else) to be mounted on I(path).
```

Comment:
Could you please add a changelog entry for this change - [here](https://github.com/ansible/ansible/tree/devel/changelogs/fragments)

Reply:
I don't think it is needed, as it is not really a change, just a doc clarification. I could not find any other changelog entry for doc clarification.

Reply:
If you do insist on the changelog entry, would this be good enough?

```
---
minor_changes:
  - mount - documented that NFS and other non-device-based filesystems can be used
```

Reply:
Sorry, I should have been more clear. Actually, I meant for overall change, not just this line. Adding a changelog entry will help to backport this change to 2.9 release.

Reply:
OK, understood, will prepare it in a few minutes.

Reply:
Done. For the record, the NFS breakage was specific to the development version, and I am not 100% convinced that the fix for #59183 (i.e. better error handling in the case of a failed mount) should be backported to stable versions.

Reply:
OK, let me use different words, hopefully more clear ;)

The original (bad) fix for #59183 was not deemed important enough to be backported, so why do you treat my (hopefully better) fix differently?